third_party/arcs/proto/manifest.proto

```diff
@@ -111,11 +109,9 @@ message TypeProto {
     SingletonTypeProto singleton = 7;
     CountTypeProto count = 8;
     ListTypeProto list = 9;
-    NullableTypeProto nullable = 13;
   }
+  bool optional = 10;
```

Comment:
Will taking over `proto` field number 10 for `optional` instead of whatever it was used for in the past cause incompatibilities with old Arcs `proto`s we might analyze? Maybe there aren't enough of them for it to matter, but it might be safer to keep the `reserved 10` and give the new `optional` a new number? Similar concerns for Enum member deletions.

Reply:
Or is the diff misleading me? Is this the `proto` file as it appears in the original Arcs repo?

Reply:
I noticed this just after sending the PR for review. It seems like the webpack bundle was created off of an earlier commit in the Arcs repo. I am trying to build one off of HEAD. I will hold off merging this until I resolve this one way or another. 

In any case, I don't expect us to use protos created outside of our repo. So, it shouldn't be a big deal even if we use a slightly older version of the proto.

Reply:
OK, whichever way this resolves, this sounds fine to me.

Reply:
Let us get this in. We can update the protos and the webpack in a subsequent PR.

Reply:
SGTM.